server/athenian/api/controllers/status_controller.py

```diff
@@ -111,12 +114,13 @@ def _after_response(request: web.Request,
 
 @web.middleware
 async def instrument(request: web.Request, handler) -> web.Response:
-    """Middleware to count requests and record the elapsed time."""
+    """Middleware to count requests, record the elapsed time and track features flags."""
     start_time = time.time()
     request.app["request_in_progress"] \
         .labels(__package__, __version__, request.path, request.method) \
         .inc()
     request.app["db_elapsed"].set(defaultdict(float))
+    request.app["metrics-calculator"].set(defaultdict(str))
```

Comment:
Can you please use underscore vs. dash

```suggestion
    request.app["metrics_calculator"].set(defaultdict(str))
```

Reply:
right, sorry, I overlooked it